velox/vector/tests/VectorTest.cpp

```diff
@@ -445,7 +445,6 @@ class VectorTest : public testing::Test, public test::VectorTestBase {
     auto base = decoded.base();
     auto nulls = decoded.nulls();
     auto indices = decoded.indices();
```

Comment:
Do we have a dedicated test case that triggers this crash? Thanks!

Reply:
Yes, we do now. VectorTest::row goes through that path. It used to make Constant null vector of size 20, but I changes it to 123.

```
// Add constant wrapping using null row.
    if (source->mayHaveNulls() && !source->isLazy()) {
      int32_t firstNull = 0;
      for (; firstNull < sourceSize; ++firstNull) {
        if (source->isNullAt(firstNull)) {
          break;
        }
      }
      constant = BaseVector::wrapInConstant(firstNull + 123, firstNull, source);
      testCopy(constant, level - 1);
    }
```